### ISSUE TITLE
Bug for varargs

### ISSUE THREAD

**Issue by karpierz** [ninia/jep#148]
- Jep Distribution and Version: 3.1.8
- OS Platform, Distribution, and Version: All
- Python Distribution and Version: 3.6
- Java Distribution and Version: Oracle Java 1.8

Bug in variable arguments handling.

Trivial patch:
```
--- pyjmethod.c.org	2018-07-13 18:43:24.000000000 +0200
+++ pyjmethod.c	2018-08-01 09:47:26.025288800 +0200
@@ -250,7 +250,7 @@
             return NULL;
         }
         /* The last argument will be handled as varargs, so not a normal arg */
-        lenJArgsNormal -= lenJArgsExpected - 1;
+        lenJArgsNormal = lenJArgsExpected - 1;
     } else {
         /* No varargs, all args are normal */
         lenJArgsNormal = lenJArgsExpected;
```

Best regards,
Adam Karpierz

**Comment by bsteffensmeier** [comment]
Good catch, I'll work on getting better unit tests for this and putting your fix in dev_3.8. This is simple enough and severe enough to justify a new point release for 3.8. We will wait a few weeks to see if any more 3.8 bugs pop up and then make an official release of 3.8.2 with this fix.

**Comment by bsteffensmeier** [comment]
Fix is in dev_3.8 with c4af3c39

**Comment by bsteffensmeier** [comment]
Just released 3.8.2 with this fix.
